src/test/duplicate-email.ts

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+import chai from 'chai';
+import { UserInput } from './../resolvers';
+import { dataSource } from '../data-source';
+import { User } from '../user';
+
+describe('Testing returned Custom Error', () => {
+  it('should return duplicate email error', async () => {
```

Comment:
No geral a gente separa o teste de uma query/mutation, com todos os seus casos de teste, em um arquivo com um describe só. Então esse cara aqiu poderia ser um "it" adicional no arquivo do teste de criação de usuário.